DataConnectors/ALC-WebCTRL/Connector_WindowsEvents_WebCTRL.json

```diff
@@ -0,0 +1,102 @@
+    "descriptionMarkdown": "You can stream the audit logs from the WebCTRL SQL server hosted on Windows machines connected to your Azure Sentinel. This connection enables you to view dashboards, create custom alerts and improve investigation. This gives insights into your Industrial Control Systems that are monitored or controlled by the WebCTRL BAS application.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Events",
+            "baseQuery": "Event\n| where Source == \"ALCWebCTRL\""
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Total warnings and errors raised by the application",
+            "query": "Event\n| where Source == \"ALCWebCTRL\"\n| where EventLevel in (1,2,3)"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Events",
+            "lastDataReceivedQuery": "Event\n| where Source == \"ALCWebCTRL\"\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Event\n| where Source == \"ALCWebCTRL\"\n| summarize LastLogReceived = max(TimeGenerated)\n| project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "1. Install and onboard the Microsoft agent for Windows.",
+            "description": "Learn about [agent setup](https://docs.microsoft.com/en-us/services-hub/health/mma-setup) and [windows events onboarding]() in the documentation.](https://docs.microsoft.com/en-us/azure/azure-monitor/agents/data-sources-windows-events) \n\n You can skip this step if you have already installed the Microsoft agent for Windows"
```

Comment:
Please remove locale from url.

Reply:
Removed the locale from url in commit [915c6](https://github.com/Azure/Azure-Sentinel/pull/3335/commits/915c6eac5259cea8fc2a7e5818063d87e400fe4b)